pkg/action/cl002/ac001/crs.go

```diff
@@ -0,0 +1,77 @@
+package ac001
+
+import (
+	"github.com/giantswarm/apiextensions/pkg/apis/infrastructure/v1alpha2"
+	"github.com/giantswarm/microerror"
+
+	"github.com/giantswarm/awscnfm/pkg/key"
+	"github.com/giantswarm/awscnfm/pkg/release"
+)
+
+func newCRs(host string) (v1alpha2.ClusterCRs, v1alpha2.NodePoolCRs, error) {
+	var err error
+
+	var releaseComponents map[string]string
+	{
+		c := release.Config{
+			Branch: "aws-release-12-0-0-2",
+		}
+
+		releaseCollection, err := release.New(c)
+		if err != nil {
+			return v1alpha2.ClusterCRs{}, v1alpha2.NodePoolCRs{}, microerror.Mask(err)
+		}
+
+		releaseComponents = releaseCollection.ReleaseComponents(key.Release)
+	}
+
+	var crs v1alpha2.ClusterCRs
+	{
+		c := v1alpha2.ClusterCRsConfig{
+			Credential:        key.Credential,
+			Domain:            key.DomainFromHost(host),
+			Description:       explainerCommand,
+			Owner:             key.Organization,
+			Region:            key.RegionFromHost(host),
+			ReleaseComponents: releaseComponents,
+			ReleaseVersion:    key.Release,
+		}
+
+		crs, err = v1alpha2.NewClusterCRs(c)
+		if err != nil {
+			return v1alpha2.ClusterCRs{}, v1alpha2.NodePoolCRs{}, microerror.Mask(err)
+		}
+	}
+
+	var azs []string
+	{
+		azs, err = key.AZsFromRegion(key.RegionFromHost(host))
```

Comment:
fetches all AZ's from a certain region, this is needed for the node pool creation. We take the first AZ.